Firebase/android/app/src/main/AndroidManifest.xml

```diff
@@ -32,6 +31,22 @@
 
     <activity android:name=".user.UsersActivity" />
 
+    <activity android:name=".welcome.WelcomeActivity">
+      <intent-filter>
+        <action android:name="android.intent.action.VIEW" />
+        <category android:name="android.intent.category.DEFAULT" />
+        <category android:name="android.intent.category.BROWSABLE" />
+        <data
+          android:host="novoda.com"
+          android:pathPrefix="/bonfire/welcome"
+          android:scheme="https" />
+        <data
+          android:host="novoda.com"
+          android:pathPrefix="/bonfire/welcome"
+          android:scheme="http" />
```

Comment:
Do we definitely need two `<data>` tags? I presume we do but it would be cooler if we could do `<android:scheme="http|https">` or something